pythonforandroid/toolchain.py

```diff
@@ -588,6 +676,25 @@ def warn_on_deprecated_args(self, args):
         Print warning messages for any deprecated arguments that were passed.
         """
 
+        # Output warning if setup.py is present and neither --ignore-setup-py
+        # nor --use-setup-py was specified.
+        if getattr(args, "private", None) is not None and \
+                (os.path.exists(os.path.join(args.private, "setup.py") or
+                 os.path.exists(os.path.join(args.private, "pyproject.toml")))
+                ):
+            if not getattr(args, "use_setup_py", False) and \
+                    not getattr(args, "ignore_setup_py", False):
+                warning("  **** FUTURE BEHAVIOR CHANGE WARNING ****")
```

Comment:
Feels like I already saw that somewhere else :wink:

Reply:
oh wow I just noticed this and took it out. yeah it was in twice :laughing: resolving since that is fixed now!